core/src/main/java/nl/nn/adapterframework/senders/IbisLocalSender.java

```diff
@@ -227,12 +227,12 @@ private String getActualServiceName(PipeLineSession session) throws SenderExcept
 	@Override
 	public SenderResult sendMessage(Message message, PipeLineSession session) throws SenderException, TimeoutException {
 		SenderResult result;
-		try (PipeLineSession context = new PipeLineSession()) {
+		try (PipeLineSession nestedSession = new PipeLineSession(session)) {
```

Comment:
Copying original session into the nested session is the actual fix for the issue.